SplitIt/app/src/main/java/project/cs495/splitit/ReceiptViewActivity.java

```diff
@@ -421,8 +420,7 @@ public void onDataChange(DataSnapshot dataSnapshot) {
                 if (masterReceiptSubtotal == 0) {
                     masterReceiptSubtotal = receipt.getPrice();
                 }
-                float percentOfSubtotal = subtotal / masterReceiptSubtotal;
-                userReceipt.setTax(masterReceiptTax * percentOfSubtotal);
+                userReceipt.setTax(receipt.getTax());
```

Comment:
the tax stores the dollar amount, not the rate. Thus, we want to multiply that dollar amount by the portion of the subtotal accounted for by the assigned user. I think this should be reverted.